examples/js/loaders/GLTFLoader.js

```diff
@@ -277,9 +277,9 @@ THREE.GLTFLoader = ( function () {
 		var extension = ( json.extensions && json.extensions[ EXTENSIONS.KHR_LIGHTS_PUNCTUAL ] ) || {};
 		var lights = extension.lights || {};
 
-		for ( var lightId in lights ) {
+		for ( var lightIndex = 0; lightIndex < lights.length; ++ lightIndex ) {
```

Comment:
nit: I prefer using simple loop var name `i` here for the code style consistency in this file unless this's your intentional. At least, `lightIndex ++` would be better for the consistency?

Reply:
Done ✅